tests/TestCase.php

```diff
@@ -39,4 +46,9 @@ protected function resolveApplicationHttpKernel($app)
         $app->singleton(Kernel::class, HttpKernel::class);
     }
 
+    public function setDefaultRoutes()
+    {
+        Route::get('projects', 'Wnx\LaravelStats\Tests\Stubs\Controllers\ProjectsController@index');
```

Comment:
it might be better to explicitly register this route inside the test method that tests this behaviour.

similar to https://github.com/stefanzweifel/laravel-stats/blob/master/tests/ClassifierTest.php#L102